internal/services/lighthouse/lighthouse_definition_resource.go

```diff
@@ -99,6 +100,76 @@ func resourceLighthouseDefinition() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"eligible_authorization": {
+				Type:     pluginsdk.TypeSet,
+				Optional: true,
+				Elem: &pluginsdk.Resource{
```

Comment:
Are there a maximum number of elements this Set allows in the API?

Reply:
I didn't find the maximum number from anywhere. The [`autorization`](https://github.com/hashicorp/terraform-provider-azurerm/blob/8f106f76d3af98a79f29d53c74cfd19adaa0e183/internal/services/lighthouse/lighthouse_definition_resource.go#L61) previously added also didn't mention the maximum number.